### ISSUE TITLE
Some public symbols in public headers might not be public

### ISSUE THREAD

**Issue by jan-cerny** [OpenSCAP/openscap#938]
Before we release OpenSCAP 1.3.0 we need to review our public functions, because it might turn out that some of the public functions shouldn't be public, or some of them are in wrong header files. 

For example https://github.com/OpenSCAP/openscap/blob/maint-1.2/src/OVAL/probes/SEAP/public/sm_alloc.h#L38

I think it is important to do this review because after we release 1.3.0 we will have to maintain the public API again and it will be forbidden to break it.

**Comment by yuumasato** [comment]
@jan-cerny I believe this should a blocker then

**Comment by jan-cerny** [comment]
@yuumasato Do you think that this blocks only the final release or it blocks the alpha release as well?

I think it would be very useful to have this finished in alpha, because then we can catch possible problems with other components soon.

**Comment by yuumasato** [comment]
@jan-cerny I agree with you. I even think that alpha can be a bit bold on removing symbols, and beta more conservative and fixing problems with symbols.

**Comment by jan-cerny** [comment]
It doesn't necessarily block the first alpha release.

**Comment by evgenyz** [comment]
We had our chance with `1.4` but probably we won't be able to work on that ever. Closing.
